js/main.js

```diff
@@ -53,14 +53,22 @@ window.search_data_loaded = function (search_data) {
 				resultsContent = "<p>No results found.</p>";
 				accessibleSummaryContent = "No results found."
 			}
+
 			$("#results-placeholder").html(resultsContent);
-			resultsElement.show();
-			contentElement.hide();
 			accessibleElement.html(accessibleSummaryContent);
+
+			if (resultsElement.is(":hidden")) {
+				// Display page title dynamically to avoid confusing bots that search
+				// for metadata
+				resultsElement.prepend("<h1>Search results</h1");
+				resultsElement.show();
+				contentElement.hide();
+			}
 		} else {
 			// When the search input is empty, hide the result page and display the
 			// current page. Also clear the accessibility notification area.
 			resultsElement.hide();
+			resultsElement.children("h1").remove();
```

Comment:
`resultsElement.empty()`

Reply:
I don't think I can do that because if I do, I lose `#results-placeholder` and `#accessible-search-summary`.
But that might be moot if I do that in `resultsContent` per your other comment.

Reply:
There's no visual difference if h1 is inside or outside the results div

Reply:
Don't think we understood each other on that one: what I was saying is that I believe `resultsElement.empty()` would clear everything in it, including the placeholders for search results and SR summary. Maybe I misunderstood though.
Agree that there is no visual difference though.

Reply:
Yeah I misread the comment context, was thinking you were talking about the h1.